### PR TITLE
Fix wasm_proxied_mount_css to not reuse an existing style element

### LINKED ISSUE #7694: Does GradioLite support Gradio custom theme?
### Describe the bug

Trying to develop a custom theme with Gradio lite but it doesn't seem to work.

the only theme related doc that I can find on Gradio Lite is

> Note: you can add the theme attribute to the <gradio-lite> tag to force the theme to be dark or light (by default, it respects the system theme). E.g.

which is not what I'm expecting. thx

(searched Gradio Lite theme in the Issue queue. Nothing found, so raising a bug here)

### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Reproduction

```python
import gradio as gr


from typing import Iterable
import gradio as gr
from gradio.themes.base import Base
from gradio.themes.utils import colors, fonts, sizes
import time


class Seafoam(Base):
    def __init__(
        self,
        *,
        primary_hue: colors.Color | str = colors.emerald,
        secondary_hue: colors.Color | str = colors.blue,
        neutral_hue: colors.Color | str = colors.blue,
        spacing_size: sizes.Size | str = sizes.spacing_md,
        radius_size: sizes.Size | str = sizes.radius_md,
        text_size: sizes.Size | str = sizes.text_lg,
        font: fonts.Font
        | str
        | Iterable[fonts.Font | str] = (
            fonts.GoogleFont("Quicksand"),
            "ui-sans-serif",
            "sans-serif",
        ),
        font_mono: fonts.Font
        | str
        | Iterable[fonts.Font | str] = (
            fonts.GoogleFont("IBM Plex Mono"),
            "ui-monospace",
            "monospace",
        ),
    ):
        super().__init__(
            primary_hue=primary_hue,
            secondary_hue=secondary_hue,
            neutral_hue=neutral_hue,
            spacing_size=spacing_size,
            radius_size=radius_size,
            text_size=text_size,
            font=font,
            font_mono=font_mono,
        )
        super().set(
            body_background_fill="repeating-linear-gradient(45deg, *primary_200, *primary_200 10px, *primary_50 10px, *primary_50 20px)",
            body_background_fill_dark="repeating-linear-gradient(45deg, *primary_800, *primary_800 10px, *primary_900 10px, *primary_900 20px)",
            button_primary_background_fill="linear-gradient(90deg, *primary_300, *secondary_400)",
            button_primary_background_fill_hover="linear-gradient(90deg, *primary_200, *secondary_300)",
            button_primary_text_color="white",
            button_primary_background_fill_dark="linear-gradient(90deg, *primary_600, *secondary_800)",
            slider_color="*secondary_300",
            slider_color_dark="*secondary_600",
            block_title_text_weight="600",
            block_border_width="3px",
            block_shadow="*shadow_drop_lg",
            button_shadow="*shadow_drop_lg",
            button_large_padding="32px",
        )


seafoam = Seafoam()


def greet(name):
    return "Hello " + name + "!"

with gr.Blocks(theme=seafoam) as demo:
    textbox = gr.Textbox(label="Name")
    slider = gr.Slider(label="Count", minimum=0, maximum=100, step=1)
    with gr.Row():
        button = gr.Button("Submit", variant="primary")
        clear = gr.Button("Clear")
    output = gr.Textbox(label="Output")

    def repeat(name, count):
        time.sleep(3)
        return name * count

    button.click(repeat, [textbox, slider], output)

if __name__ == "__main__":
    demo.launch()


```
Copy paste the above to https://www.gradio.app/playground and wait for the preview

### Screenshot

image

I would expect: 
Screenshot 2024-03-14 at 11 25 24

### Logs

_No response_

### System Info

```shell
gradio lite
```

### Severity

I can work around it

### PR DESCRIPTION
**Pull request by whitphx**
## Description
Fixes #7694

See this comment about this PR -> https://github.com/gradio-app/gradio/pull/7709#discussion_r1525824675

### CODE REVIEW

File: js/app/src/lite/css.ts [hunk 1]
```diff
@@ -35,10 +35,11 @@ export async function wasm_proxied_mount_css(
 	});
 	const css = new TextDecoder().decode(response.body);
 
-	const existing_link = document.querySelector(
+	// Gradio Lite can be reloaded without refreshing the page, so we need to remove the existing style element if it exists.
+	const existing_style = document.querySelector(
 		`style[data-wasm-path='${url_string}']`
 	);
-	if (existing_link) return;
```
Comment (whitphx) [inline js/app/src/lite/css.ts:41]: This was implemented after the non-Wasm ver. which skips mounting the CSS if there is always a mounted element as below.
https://github.com/gradio-app/gradio/blob/28342a204001c98b86628635582fcf54aa0855aa/js/app/src/css.ts#L17-L19

However, Lite is different from the normal one in that the app can be executed repeatedly and the frontend can be refreshed many times, so this design was not correct and the requested CSS must be mounted, replacing the old one.

**Comment by whitphx** [comment]
Even this change, the playground still has a problem that the background is not styled correctly.
Will fix it as well.

Playground: CleanShot 2024-03-15 at 15 48 58@2x
Expected: 
CleanShot 2024-03-15 at 15 52 10@2x

**Comment by whitphx** [comment]
The reason why the background theme is not applied is this:
https://github.com/gradio-app/gradio/blob/28342a204001c98b86628635582fcf54aa0855aa/js/app/src/Index.svelte#L259-L261
Because `handle_darkmode()` is not called here due to the if-condition, `darkmode()` is not called finally, then the background-update code below is not executed.
https://github.com/gradio-app/gradio/blob/28342a204001c98b86628635582fcf54aa0855aa/js/app/src/Index.svelte#L238

**Review by aliabd — APPROVED** [review]
Thanks @whitphx everything lgtm. 

there's a funny effect on playground where the theme also affects the code component. 

Screen Shot 2024-03-18 at 1 31 25 AM

but i actually think that's fine

**Comment by whitphx** [comment]
@aliabd Thanks!
Let's discuss about the playground theming issue if you find it not good later 👍

**Pull request merged**